### ISSUE TITLE
The chart is not loaded in the Blazor Web App

### ISSUE THREAD

**Issue by ertan2002** [apexcharts/Blazor-ApexCharts#347]
I have a new blazor web app (it is a new template in .net 8.0 )and installed the latest version of the blazor-apexCharts 2.1.0
and unfortunately I cannot display the chart. 

Then I just created a new web assembly app and used the same code and it worked.

Is there any special settings for the blazor web app? or maybe blazor-apexCharts does not support this new template yet?

a screenshot is here from the web assembly project

![image](https://github.com/apexcharts/Blazor-ApexCharts/assets/1535195/95d7a30c-b2dd-4b68-9298-7141326abaa8)

Here is my sample code

`    
  @using ApexCharts;
  @using System.Runtime.InteropServices.JavaScript
   @page "/testchart"

           <h3>TestChart</h3>

        <ApexChart TItem="StockPrice"
                   XAxisType="XAxisType.Datetime"
                   Options=chartOptions>

            <ApexCandleSeries TItem="StockPrice"
                              Items="StockPrices"
                              Name="Apex Stock Price"
                              XValue="@(e => e.Date.ToUnixTimeMilliseconds())"
                              Open="@(e => e.Open)"
                              High="@(e => e.High)"
                              Low="@(e => e.Low)"
                              Close="@(e => e.Close)"
                              OrderByDescending="e => e.X"
                              Stroke="@(new SeriesStroke { Width = 1 })"/>

            <ApexPointSeries Items=LineData
                             TItem="StockPrice"
                             Name="Line"
                             SeriesType="SeriesType.Line"
                             XValue="@(e => e.Date.ToUnixTimeMilliseconds())"
                             YValue="@(e => e.High)"
                             OrderByDescending="@(e => e.X)"
                             Stroke="@(new SeriesStroke { Width = 3, DashSpace = 3, Color = "#000000" })"
                             ShowDataLabels="false"
            />
                  

        </ApexChart>

        @code {

            public List<StockPrice> StockPrices { get; set; } = GetStockPrices();
            private ApexChartOptions<StockPrice> chartOptions = new ApexChartOptions<StockPrice>();

            protected override void OnInitialized()
            {
                chartOptions.Annotations = new Annotations
                {
                    Points = new List<AnnotationsPoint>()
                    {
                        new AnnotationsPoint()
                        {
                           X = (new DateTime(2020, 09, 01)).ToUnixTimeMilliseconds(),
                            Y = 130,
                            Label = new Label(){ Text = "Swing High",Style = new Style {Color = "black", FontSize = "16px", FontWeight ="bold"},
                                BorderColor = "blue",
                                Position = LabelPosition.Left,
                                TextAnchor = TextAnchor.Start,
                                // OffsetY = 12,
                                // OffsetX = 20
                            },
                            Marker = new AnnotationMarker
                            {
                                Size = 8
                            },
                        }
                    }     
                };
            }

            private static List<StockPrice> GetStockPrices()
            {
                return new List<StockPrice>()
                {
                    new StockPrice() { Date = new DateTime(2020, 09, 01), Open = 100, Close = 120, High = 130, Low = 90 },
                    new StockPrice() { Date = new DateTime(2020, 09, 02), Open = 110, Close = 105, High = 115, Low = 100 },
                    new StockPrice() { Date = new DateTime(2020, 09, 03), Open = 105, Close = 115, High = 125, Low = 100 },
                    new StockPrice() { Date = new DateTime(2020, 09, 04), Open = 100, Close = 80, High = 110, Low = 72 },
                    new StockPrice() { Date = new DateTime(2020, 09, 05), Open = 80, Close = 90, High = 95, Low = 65 },
                    new StockPrice() { Date = new DateTime(2020, 09, 06), Open = 90, Close = 65, High = 70, Low = 75 },
                    new StockPrice() { Date = new DateTime(2020, 09, 07), Open = 65, Close = 55, High = 55, Low = 50 },
                    new StockPrice() { Date = new DateTime(2020, 09, 08), Open = 55, Close = 45, High = 53, Low = 33 },
                    new StockPrice() { Date = new DateTime(2020, 09, 09), Open = 40, Close = 40, High = 48, Low = 35 },
                    new StockPrice() { Date = new DateTime(2020, 09, 10), Open = 40, Close = 70, High = 80, Low = 30 },
                    new StockPrice() { Date = new DateTime(2020, 09, 11), Open = 70, Close = 120, High = 130, Low = 90 },
                    new StockPrice() { Date = new DateTime(2020, 09, 12), Open = 110, Close = 105, High = 115, Low = 100 },
                    new StockPrice() { Date = new DateTime(2020, 09, 13), Open = 105, Close = 115, High = 125, Low = 100 },
                    new StockPrice() { Date = new DateTime(2020, 09, 14), Open = 100, Close = 80, High = 110, Low = 72 },
                    new StockPrice() { Date = new DateTime(2020, 09, 15), Open = 80, Close = 90, High = 95, Low = 65 },
                    new StockPrice() { Date = new DateTime(2020, 09, 16), Open = 90, Close = 65, High = 70, Low = 75 },
                    new StockPrice() { Date = new DateTime(2020, 09, 17), Open = 65, Close = 55, High = 55, Low = 50 },
                    new StockPrice() { Date = new DateTime(2020, 09, 18), Open = 55, Close = 45, High = 53, Low = 33 },
                    new StockPrice() { Date = new DateTime(2020, 09, 19), Open = 40, Close = 40, High = 48, Low = 35 },
                    new StockPrice() { Date = new DateTime(2020, 09, 20), Open = 40, Close = 70, High = 80, Low = 30 }
                };
            }

            public List<StockPrice> LineData { get; set; } = new List<StockPrice>
            {
                new StockPrice() { Date = new DateTime(2020, 09, 01), Open = 100, Close = 120, High = 130, Low = 90 },
                new StockPrice() { Date = new DateTime(2020, 09, 03), Open = 105, Close = 115, High = 130, Low = 100 }
            };
            
            public class StockPrice
            {
                public System.DateTime Date { get; set; }
                public decimal Open { get; set; }
                public decimal Close { get; set; }
                public decimal High { get; set; }
                public decimal Low { get; set; }            
            }
        }
`

and here the html code of the web app

It seems that an error had occurred but in the console I don't see any error

![image](https://github.com/apexcharts/Blazor-ApexCharts/assets/1535195/fbc43376-f8c7-46b4-8502-1ee3826c93ba)

**PS:** I tried with https, http and IIS Express, the result is same
**PS 2:** I added @using Blazor_ApexCharts; in the _Imports.razor of the both projects

Now I realize something that some files in the web app are not loaded or not seen, but in the webassembly app seems ok

![image](https://github.com/apexcharts/Blazor-ApexCharts/assets/1535195/f3e94582-8391-4d0a-9d7e-8227a2a7a311)

**Comment by joadan** [comment]
Hello,

Thanks for reporting. 
I will investigate asap.

**Comment by dschuesae** [comment]
For me it works, when calling RenderAsync.

**Comment by joadan** [comment]
Hello,

Not sure if I'm missing something here but I have added two simple sample apps and both seems to be working.

https://github.com/joadan/WebAppChartsServer
https://github.com/joadan/WebAppChartsWasm

**Comment by ertan2002** [comment]
> Hello,
> 
> Not sure if I'm missing something here but I have added two simple sample apps and both seems to be working.
> 
> https://github.com/joadan/WebAppChartsServer https://github.com/joadan/WebAppChartsWasm

Hi again, thank you for your help.

I found out the issue by comparing with your code.

In my TestChart.razor

**@rendermode InteractiveServer** this line was missing. 
![image](https://github.com/apexcharts/Blazor-ApexCharts/assets/1535195/4203b767-c0e3-429f-a51a-8e8619580990)

After I added this line, then it worked. 

It might be helpful to highlight this issue in the documentation for a server project. Informing users about the necessity of adding 'rendermode' to their, it could help.
